### PR TITLE
Add Symfony 4 support

### PR DESCRIPTION
**Pull request by welcoMattic**

### CODE REVIEW

**Comment by jordisala1991** [comment]
Can you please adapt .travis.yml from the TranslationFormBundle?

There are some improvements that should be ported here

File: .travis.yml [hunk 1]
```diff
@@ -1,25 +1,60 @@
...
 
 install:
-    - composer --prefer-source $COMPOSER_EXTRA update
+  - mkdir --parents $HOME/bin
+  - wget https://github.com/satooshi/php-coveralls/releases/download/v1.0.1/coveralls.phar --output-document=$HOME/bin/coveralls
+  - chmod u+x $HOME/bin/coveralls
+  - composer update --prefer-dist --no-interaction --prefer-stable $COMPOSER_FLAGS
 
 script:
     - vendor/bin/phpunit Tests/
```
Comment (jordisala1991) [inline .travis.yml]: use phpunit bridge instead of phpunit please

**Comment by jordisala1991** [comment]
And you probably need to add phpunit.xml.dist to be able to execute test without telling what folder to use

**Comment by webda2l** [comment]
Thank @welcoMattic. I think about replace your PR by a more deeper https://github.com/a2lix/AutoFormBundle/pull/7
What's your PHP version and Symfony version BTW ?

**Comment by welcoMattic** [comment]
Ok to replace ;)

I'm working with PHP 7.1 and Symfony 4.x

**Comment by webda2l** [comment]
Ok fine, so you have the GO to test the new beta release. :)

**Pull request closed without merging**